src/main/drivers/camera_control.c

```diff
@@ -28,11 +28,11 @@
 #include "config/parameter_group_ids.h"
 
 #if defined(STM32F40_41xxx)
-#define CAMERA_CONTROL_TIMER_MHZ   84
+#define CAMERA_CONTROL_TIMER_MHZ   MHZ_TO_HZ(84)
```

Comment:
This should be renamed `CAMERA_CONTROL_TIMER_HZ`, since it is now a value in Hz, not MHz.